### ISSUE TITLE
Feature Request: TVGuide on Android Display

### ISSUE THREAD

**Issue by GoogleCodeExporter** [pot8oe/mythmote#45]
```
What steps will reproduce the problem?
1.Pressing TVGuide Button on Navigation Tab opens TVGuide on MythTV Output 
Device (i.e., TV).
2.
3.

What is the expected output? What do you see instead?
Same.

What version of the product are you using? On what operating system?
Current version available in Market.

Please provide any additional information below.
If possible I'd like a setting option to display the TVGuide on the remote 
(android device)'s display rather than calling up the TVGuide on the MythTV 
display.  

In other words, I'd like to have the TV output remain the same while the 
android device displays the contents of the guide.

Just a thought.  :)

kev.
```

Original issue reported on code.google.com by `yu210...@gmail.com` on 8 Nov 2010 at 12:47
